tests/unittest/_torch/sampler/test_beam_search.py

```
@@ -609,6 +735,125 @@ def test_beam_search_sampling_batch_basic():
+        beam_width_in=1,
+        beam_width_out=beam_width,
+        beam_search_args=continuous_metadata,
+        temperature=temperature,
+        return_probs=False,
+    )
+    first_gen_cum_log_probs = continuous_metadata.cum_log_probs[
+        seq_slots, :beam_width].clone()
+    assert first_tokens.shape == (batch_size, beam_width)
+
+    continuous_metadata.seq_lens = torch.full((batch_size, ),
+                                             prompt_len + 1,
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)